.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
```

Comment:
I'd name this something more descriptive like `docker`. 
We also have a common workflow for this that you can just call. 

See node's example
https://github.com/celestiaorg/celestia-node/blob/main/.github/workflows/docker-build-publish.yml